test/common/http/http1/codec_impl_test.cc

```diff
@@ -682,7 +721,41 @@ TEST_F(Http1ServerConnectionImplTest, ChunkedResponse) {
 
   Buffer::OwnedImpl data("Hello World");
   response_encoder->encodeData(data, true);
-  EXPECT_EQ("HTTP/1.1 200 OK\r\ntransfer-encoding: chunked\r\n\r\nb\r\nHello World\r\n0\r\n\r\n",
+
+  EXPECT_EQ("HTTP/1.1 200 OK\r\ntransfer-encoding: chunked\r\n\r\nb\r\nHello "
+            "World\r\n0\r\n\r\n",
+            output);
+}
+
+TEST_F(Http1ServerConnectionImplTest, ChunkedResponseWithTrailers) {
```

Comment:
I'd like to see unit tests of parsing invalid input trailers, as well as overly large trailers.

Reply:
For overly large trailers it gets tested in HttpIntegrationTest::testLargeRequestTrailers.

Reply:
I have unit tests for both the scenarios you listed. I'll push them up soon.